_docs/_latest/configure/scanner/descriptions.md

```diff
@@ -166,3 +166,20 @@ than the max age.
 For examples of how to define scanner rules for your service account keys, see the
 [`service_account_key_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/stable/rules/service_account_key_rules.yaml)
 file.
+
+
+## External Project Access Scanner
+
+Data exfiltration is a common threat in cyber security and in the Cloud.  The External Project Access Scanner mitigates this by identifying users who have access to projects where the project ancestor hierarchy does not include  whitelisted parent.  
+
+### Scanner Operation
+Forseti scanners follow the pattern of using an model which is derived from the inventory.  A model details how GCP resources are related.  A scanner will analyze a model against user defined rules.  Because a model is stored in the Forseti database, the access of the model is relatively fast.
```

Comment:
This scanner operation section doesn't seem to fit here.  I think we can remove it.

Reply:
modified as suggested